cmd/juju/storage/list_test.go

```diff
@@ -56,20 +56,39 @@ func (s *ListSuite) TestListYAML(c *gc.C) {
 		c,
 		[]string{"--format", "yaml"},
 		`
-db-dir/1000:
-  storage: db-dir
-  owner: postgresql/0
-  location: /srv/data
-  available-size: 1
-  total-size: 1024
-  tags:
-  - tests
-  - well
-  - maybe
-shared-fs/0:
-  storage: shared-fs
-  owner: transcode/0
-  location: /srv
+postgresql/0:
+  db-dir/1000:
+    storage: db-dir
+    location: /srv/data
+    available-size: 1
+    total-size: 1024
+    tags:
+    - tests
+    - well
+    - maybe
+transcode/0:
+  shared-fs/0:
+    storage: shared-fs
+    location: /srv
+`[1:],
+	)
+}
+
+func (s *ListSuite) TestListOwnerStorageIdSort(c *gc.C) {
+	s.mockAPI.lexicalChaos = true
+	s.assertValidList(
+		c,
+		nil,
+		// Default format is tabular
+		`
+[Storage]   
+ID          OWNER        SIZE      LOCATION  
```

Comment:
My preference would be to see OWNER listed first, and then ID (since we are sorting by OWNER)